content/tutorial-deep-learning-on-mnist.md

```diff
@@ -74,7 +74,6 @@ download it.
 
 ```{code-cell} ipython3
 :tags: [remove-cell]
-
```

Comment:
This blank line separates the cell metadata (the tags in this case) from the actual code in the cell. I'm actually not sure if it's *required* by the myst-nb text-based notebook format, but I personally like keeping them separated by a blank line.

Alternatively, I believe the format supports a different format to make this separation more explicit, something like:
````
```{code-cell} ipython3
---
tags: [remove-cell]
---
# Code starts here
```

Reply:
Ah I hadn't noticed that - sorry, it was a copy-past mistake!